lib/cocoapods-rome/post_install.rb

```diff
@@ -50,6 +52,17 @@ def xcodebuild(sandbox, target, sdk='macosx')
   Pod::UI.puts "Copying frameworks to `#{destination.relative_path_from Pathname.pwd}`"
```

Comment:
We should move this messaging below the new frameworks copying so it has the correct number